### DISCUSSION TITLE
设计：setup页面的UX

### DISCUSSION THREAD

**Discussion by JeffersonHuang** [h65wang/flutter_interview_questions#56] [Ideas]
* setup页面的UX，大伙可以在这里讨论。

**Comment by JeffersonHuang** [comment]
@zhuzhao404 @h65wang @huanghui1998hhh

**Comment by h65wang** [comment]
桥豆麻袋，容我PR一个大改动先！

**Comment by h65wang** [comment]
What do you think about this UI?
![something i saw on dribbble](https://cdn.dribbble.com/users/2567945/screenshots/5881563/media/212a47882104b11acf56d9212de9fea0.png)

**Reply by JeffersonHuang** [reply]
This look great！

**Reply by zhuzhao404** [reply]
Looks good! BTW,the correct answer should also have a checked status!

**Reply by JeffersonHuang** [reply]
Will a border green color be shown if users select the correct answer?
